bugbug/repository.py

```diff
@@ -372,6 +378,23 @@ def _transform(commit):
 
             if size is not None:
                 source_code_sizes.append(size)
+                metrics = code_analysis_server.metrics(path, after)
+                if "error" in metrics:
+                    metrics_file_count += 1
+                    metrics = metrics["spaces"]["metrics"]
+                    commit.average_cyclomatic += metrics["cyclomatic"]
+                    commit.average_halstead_unique_operands += metrics["halstead"][
+                        "unique_operands"
+                    ]
+                    commit.average_halstead_operands += metrics["halstead"]["operands"]
+                    commit.average_halstead_unique_operators += metrics["halstead"][
+                        "unique_operators"
+                    ]
+                    commit.average_halstead_operators += metrics["halstead"][
+                        "operators"
+                    ]
+                    commit.average_source_loc += metrics["loc"]["sloc"]
+                    commit.average_logical_loc += metrics["loc"]["lloc"]
```

Comment:
Are these at the file level or the function level?

Reply:
They are at file level, as discussed on IRC, let's get mean/max/min/avg complexities both at function level and at file level